### PR TITLE
[TwigBundle] Fix log message formatting in exception views

### PR DESCRIPTION
**Pull request by chalasr**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #22335
| License       | MIT
| Doc PR        | n/a

| __Before__      |
| ------------- |
| before   | 

| __After__        |
| ------------- |
| after   |

### CODE REVIEW

File: src/Symfony/Bundle/TwigBundle/Resources/views/Exception/logs.html.twig [hunk 1]
```diff
@@ -21,7 +21,7 @@
                     {{ log.channel }}
                 </td>
             {% endif %}
-            <td>{{ log.message }}</td>
+            <td>{{ log.message|format_log_message(log.context) }}</td>
```
Comment (curry684) [inline src/Symfony/Bundle/TwigBundle/Resources/views/Exception/logs.html.twig]: Is this filter available if the profiler is disabled but debug is enabled?

Comment (chalasr) [inline src/Symfony/Bundle/TwigBundle/Resources/views/Exception/logs.html.twig]: Yes it is.

**Review by curry684 — APPROVED** [review]

**Comment by curry684** [comment]
Status: reviewed

**Comment by fabpot** [comment]
Thank you @chalasr.

**Pull request merged**